woocommerce/woocommerce-android
# Support creating subscription products (simple/variable) from scratch

Closes: #10173 

NOTE: Don't panic about the number of file changes, most of them are string.xml removals. 



### Description

Enables creation subscription products (either simple or variable) from scratch in the app. 

### Testing instructions

1. Navigate to the product list and click + product button. 
2. Select simple subscription product type.
3. Add any details to the product, specifically all the subscription detail you can add. 
4. Save or publish the product. 
5. Check all the subscription data was properly sync with the API by checking the new product in `wp-admin`

🔁 The steps for a variation subscription product. 

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/2663464/77fa6fcd-6e02-4a71-bd0c-3bb82eb5c833
